.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+These style guides are copies of Google's internal style guides to
+assist developers working on Google owned and originated open source
+projects. Changes should be made to the internal style guide first and
+only then copied here.
+
+Unsolicited pull requests are not accepted and will be closed without
+comment.
+
+If this PR does anything other than update a style guide with changes
```

Comment:
My one problem with this text is that we've received meaningful QA-like feedback on the Python style guide via both issues and PRs.  typo fixes and broken links or references to internal sections that are omitted, etc.

I don't mind if people point those out.  I'm not going to merge the PR.  But sometimes it's the easiest way to communicate the flaw that we need to fix internally for an eventual push.

(otherwise i'm in agreement with the overall theme that we don't accept external contributions to the guides)

Reply:
I revised this a bit to allow for notification of minor errors.